src/interfaces/cartItem.d.ts

```diff
@@ -9,4 +9,5 @@ export interface ICartItem {
   quotationDetails: PriceBar[];
   image: string;
   isExpired: boolean;
+  constructionDrawing?: string;
```

Comment:
Is this a URL string? If yes, all the same naming of this variable/type/interface is not suitable.

Reply:
Yes I agree. I did it to align the naming on the backend. I initally used a more readable name but found that I would have to change all the relevant naming on the backend, so I use the same name. What should I do in this situation?